crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -59,18 +64,29 @@ pub struct ParsedQProperty {
 fn parse_meta_name_value(name_value: &MetaNameValue) -> Result<(Ident, Ident)> {
     let ident = name_value.path.require_ident()?.clone();
     let expr = &name_value.value;
-    let func_signature: Ident;
-
-    if let Expr::Path(path_expr) = expr {
-        func_signature = path_expr.path.require_ident()?.clone();
+    let value: Ident;
+
+    // Only cxx_name and rust_name can use string literals for values
+    if ident == "cxx_name" || ident == "rust_name" {
+        let string = expr_to_string(expr).map_err(|_| {
+            Error::new(
+                ident.span(),
+                "cxx_name and rust_name must use string values like `cxx_name = \"myName\"`!",
+            )
+        })?;
+        value = format_ident!("{}", string);
```

Comment:
seems a bit unfortunate that it ends up back as a Ident when this is a string :thinking: but would require larger changes i guess....

Reply:
Yeah it's just a prototype atm to see if we can get the correct behaviour

Reply:
On 2nd thoughts this is probably fine, as it is still identifying something, and used to fill fields in Name, but we just want the user to specify it in quotes?

Reply:
Right we still wants the quotes for the input. But yes on the Rust side this is actually turning into an Ident and on the C++ side a String as usual, so not terrible and probably fine to keep this. Was more an observation.